dashboard/test/factories.rb

```diff
@@ -1,4 +1,17 @@
 FactoryGirl.define do
+  factory :plc_enrollment_unit_assignment, :class => 'Plc::EnrollmentUnitAssignment' do
```

Comment:
Can you group all of the plc factories and place them at the end after more foundational factories like user etc.?